tests/actions/test_scan_action.py

```diff
@@ -133,6 +135,7 @@ def scan_action(
     )
 
 
+@mock.patch.dict(os.environ, {"API_KEY": "", "API_ENDPOINT": ""}, clear=True)
```

Comment:
Is there a way to do this without adding it to every single test?

Reply:
I was looking at it last Friday, but it does not work out.